userspace/falco/app/actions/process_events.cpp

```diff
@@ -435,7 +435,6 @@ static falco::app::run_result init_stats_writer(
 		falco_logger::log(LOG_WARNING, "Metrics are enabled with no output configured, no snapshot will be collected");
 	}
 
```

Comment:
I cannot comment on the right line but above (https://github.com/falcosecurity/falco/pull/2739/files#diff-3a3a5ebb7d4bf516a4c732cadcd83feabe05a5cc76a9ca9e1f55802a94b3c4a7R428), there is a check on the prometheus stats format that say 
```cpp
falco_logger::log(LOG_WARNING, "Metrics interval was passed as numeric value without Prometheus time unit, this option will be deprecated in the future");
``` 
Probably we want to turn it into an `falco::app::run_result::fatal` , WDYT?
It could be a future cleanup if we want merge this :)